pkg/proc/gdbserial/gdbserver.go

```diff
@@ -656,6 +656,14 @@ func (p *gdbProcess) EntryPoint() (uint64, error) {
 // stubs will report both the PID and executable path.
 func (p *gdbProcess) initialize(path, cmdline string, debugInfoDirs []string, stopReason proc.StopReason) (*proc.TargetGroup, error) {
 	var err error
+
+	if p.conn.undoSession != nil {
+		path, err = undoGetExePath(&p.conn)
```

Comment:
This overwrites the `path` that was passed in as an argument, which I think deserves a comment.

Reply:
:+1: